01_fib/luliangLi/main.go

```diff
@@ -0,0 +1,60 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+var out io.Writer = os.Stdout
+
+const (
+	fibNegaone = 1
+	fibZero    = 0
+	fibOne     = 1
```

Comment:
I think is an overkill to have all these constants, if the concern is to avoid magic numbers, I believe 1 or 0 do not fall into this category, I would use the values themselves.

Reply:
Thanks, and I agree, already removed the const block.